test/e2e/tests/00-begin.js

```diff
@@ -4,44 +4,76 @@ const config = require('../../../config');
 // FIXME: This should come in from a config variable
 const url = 'http://localhost:' + config.port; // eslint-disable-line
 
+// Test timeout constants
+// TODO(awong): Move to a common file usable for all tests.
+const defaultTimeout = 500;   // The normal timeout to use. For most opreations w/o a server roundtrip, this should be more than fast enough.
+const slowTimeout = 1000;     // A slow timeout incase the page is doing something complex.
+const molassesTimeout = 5000; // A really really slow timeout. This should rarely be used.
```

Comment:
Partially because eslint complains about unused variables, would using a dict like `const timeouts = { default: 500, slow: 1000, molasses: 5000};` make sense for these?